src/cloudflared.ts

```diff
@@ -0,0 +1,2410 @@
+      args: {
+        name: "value",
+        description: "Bootstrap endpoint",
+        isVariadic: true,
+      },
+    },
+    {
+      name: ["--credentials-file", "--cred-file"],
+      description:
+        "Filepath at which to read/write the tunnel credentials [$TUNNEL_CRED_FILE]",
+      args: {
+        name: "value",
+        description: "Path",
+        template: ["filepaths"],
+      },
+    },
+    {
+      name: "--region",
+      description:
+        "Cloudflare Edge region to connect to. Omit or set to empty to connect to the global region. [$TUNNEL_REGION]",
+      args: { name: "value", description: "Region" },
+    },
+    {
+      name: "--hostname",
+      description:
+        "Set a hostname on a Cloudflare zone to route traffic through this tunnel. [$TUNNEL_HOSTNAME]",
+      args: { name: "value", description: "Hostname" },
+    },
+    {
+      name: "--lb-pool",
+      description:
+        "The name of a (new/existing) load balancing pool to add this origin to. [$TUNNEL_LB_POOL]",
+      args: { name: "value", description: "Pool name" },
+    },
+    {
+      name: "--metrics-update-freq",
+      description:
+        "Frequency to update tunnel metrics (default: 5s) [$TUNNEL_METRICS_UPDATE_FREQ]",
+      args: { name: "value", description: "Frequency" },
+    },
+    {
+      name: "--tag",
+      description:
+        "Custom tags used to identify this tunnel, in format KEY=VALUE. Multiple tags may be specified  (accepts multiple inputs) [$TUNNEL_TAG]",
+      args: {
+        name: "KEY",
+        description: "Tag",
+        isVariadic: true,
+      },
+    },
+    {
+      name: "--retries",
+      description:
+        "Maximum number of retries for connection/protocol errors. (default: 5) [$TUNNEL_RETRIES]",
+      args: { name: "value", description: "Max retries" },
+    },
+    {
+      name: "--grace-period",
+      description:
+        "When cloudflared receives SIGINT/SIGTERM it will stop accepting new requests, wait for in-progress requests to terminate, then shutdown. Waiting for in-progress requests will timeout after this grace period, or when a second SIGTERM/SIGINT is received. (default: 30s) [$TUNNEL_GRACE_PERIOD]",
+      args: { name: "value", description: "Time (s)" },
+    },
+    {
+      name: "--compression-quality",
+      description:
+        "(beta) Use cross-stream compression instead HTTP compression. 0-off, 1-low, 2-medium, >=3-high. (default: 0) [$TUNNEL_COMPRESSION_LEVEL]",
+      args: {
+        name: "value",
+        description: "Compression quality",
+        suggestions: [
+          { name: "0", description: "Off" },
+          { name: "1", description: "Low" },
+          { name: ">3", description: "High" },
```

Comment:
If we insert >3 the CLI will probably reject that value.
```suggestion
          { name: "3", description: "High" },
```